.eslintrc.cjs

```diff
@@ -45,6 +45,18 @@ const config = {
       { prefer: 'type-imports' },
     ],
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/naming-convention': [
+      'warn',
```

Comment:
I define this rule as `warn` first to use this rule quickly together

Reply:
could we try to error it and just rename the type parameters if necessary? This should be non breaking ...

Reply:
I made this rule as error in [77af7b3](https://github.com/TanStack/query/pull/6684/commits/77af7b3dde36f11583e35566a4b59ebbae51f43a)
At first, this rule was applied as a warning so that it would be applied gradually and smoothly, but I think it would be better to force it as an error. Thanks!